lib/Drupal/Core/StringTranslation/TranslationWrapper.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Core\StringTranslation\TranslationWrapper.
+ */
+
+namespace Drupal\Core\StringTranslation;
+
+/**
+ * Provides a class to wrap a translatable string.
+ *
+ * This class can be used to delay translating strings until the translation
+ * system is ready. This is useful for using translation in very low level
+ * subsystems like entity definition and stream wrappers.
+ *
+ * @see \Drupal\Core\Annotation\Translation
+ */
+class TranslationWrapper {
+  use StringTranslationTrait;
```

Comment:
Did it, there might be other ways to fix this. Let me know if you're ok with this.